src/DoctrineMongoODMModule/Module.php

```diff
@@ -42,8 +41,7 @@
  * @since   0.1.0
  * @author  Tim Roediger <superdweebie@gmail.com>
  */
-class Module implements
-    BootstrapListenerInterface,
+final class Module implements
```

Comment:
@Bilge I understand your point, but in practice we don't know how people are doing and in theory this is a bc break. I can not merge it